ddlmaker.go

```diff
@@ -25,22 +25,22 @@ type DDLMaker struct {
 	Tables  []dialect.Table
 }
 
-// NewMaker return DDLMaker
-func NewMaker(conf Config) (*DDLMaker, error) {
-	dialect, err := dialect.NewDialect(conf.DB.Driver, conf.DB.Engine, conf.DB.Charset)
+// New creates a DDLMaker and returns it.
+func New(conf Config) (*DDLMaker, error) {
```

Comment:
`NewMaker` returns a `DDLMaker` which is equal to the package name of `ddlmaker`. So the suffix of `Maker` should be omitted. See https://golang.org/doc/effective_go.html#package-names :

> The importer of a package will use the name to refer to its contents, so exported names in the package can use that fact to avoid stutter. (Don't use the import . notation, which can simplify tests that must run outside the package they are testing, but should otherwise be avoided.) For instance, the buffered reader type in the bufio package is called Reader, not BufReader, because users see it as bufio.Reader, which is a clear, concise name. Moreover, because imported entities are always addressed with their package name, bufio.Reader does not conflict with io.Reader. Similarly, the function to make new instances of ring.Ring—which is the definition of a constructor in Go—would normally be called NewRing, but since Ring is the only type exported by the package, and since the package is called ring, it's called just New, which clients of the package see as ring.New. Use the package structure to help you choose good names.